### PR TITLE
DLC visualization tool

### PR DESCRIPTION
**Pull request by tdincer**
What's in this PR:
- A new schema (`dlc_report`) to compute & store plotly figures for the dlc body part predictions.
- A widget to display these figures

Screenshot 2022-11-01 at 12 53 18 PM

### CODE REVIEW

File: element_deeplabcut/model.py [hunk 1]
```diff
@@ -72,6 +73,8 @@ def activate(
         add_objects=_linking_module.__dict__,
     )
 
+    dlc_report.activate(f"{model_schema_name}_report")
```
Comment (CBroz1) [inline element_deeplabcut/model.py:76]: Should we mention this additional use of `model_schema_name` in the docstring?

File: element_deeplabcut/model_report.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    """
+    schema.activate(
+        schema_name,
+        create_schema=create_schema,
+        create_tables=create_tables,
+    )
+
+
+@schema
+class PoseEstimationReport(dj.Computed):
```
Comment (CBroz1) [inline element_deeplabcut/model_report.py:30]: Please add a class docstring

**Comment by CBroz1** [comment]
Should the scope of this PR include either of the following ... ?
1. pytests in the workflow
2. documentation on using the widget

**Comment by tdincer** [comment]
> Should the scope of this PR include either of the following ... ?
> 
> 1. pytests in the workflow
> 2. documentation on using the widget

1) The tables are computed. It's a very simple schema. We can postpone the pytest to a date after SfN.
2) Yes, it's pretty simple. The widget consists of a dropdown menu and a plot button. User first selects a result and then clicks the plot button to see a trajectory plot and the trace of the positions side by side.

File: element_deeplabcut/model_report.py (hunk 2 above)
Comment (tdincer) [reply element_deeplabcut/model_report.py:30]: Done at https://github.com/datajoint/element-deeplabcut/pull/61/commits/1b6083f3169c5d10b6ca176e73b5b94e2a23b9d0.

File: element_deeplabcut/model.py (hunk 1 above)
Comment (tdincer) [reply element_deeplabcut/model.py:76]: We don't have to. It will activated without any configuration.

**Comment by tdincer** [comment]
A caveat. The trajectory figure will respond slowly to zoom in/out processes if the number of points are huge. Right now, the plots don't have the sliders to visualize a portion of the data. This feature will be added later to mitigate the slow response problem.

File: CHANGELOG.md [hunk 3]
```diff
@@ -33,6 +37,8 @@ Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
 + Add - Drafts from a collection of precursor pipelines, including [DataJoint_Demo_DeepLabCut](https://github.com/MMathisLab/DataJoint_Demo_DeepLabCut) graciously provided by the Mathis Lab.
 + Add - Support for 2d single-animal models
 
+[0.3.0]: https://github.com/datajoint/element-deeplabcut/releases/tag/0.3.0
+[0.2.1]: https://github.com/datajoint/element-deeplabcut/releases/tag/0.2.1
```
Comment (kabilar) [inline CHANGELOG.md:41]: @CBroz1 Should we make a release of `0.2.1`?

**Comment by tdincer** [comment]
> Thanks @tdincer. Could you please add a screenshot of the widget to this pull request?

@kabilar I added a screenshot of the widget in the first comment.

File: CHANGELOG.md (hunk 3 above)
Comment (CBroz1) [reply CHANGELOG.md:41]: If we plan to merge/release here before the end of this week, I would just eliminate 0.2.1 from the changelog and move the mkdocs mention to under 0.3.0

**Comment by CBroz1** [comment]
Hi @tdincer - What should happen next here? Do we have the bandwidth to add those pytests now? Are there other features you want to add before we merge?

**Pull request closed without merging**